templates/.phpcs.xml.dist

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Plugins">
+    <description>Generally-applicable sniffs for WordPress plugins.</description>
+
+    <!-- What to scan -->
+    <file>.</file>
+    <exclude-pattern>vendor/</exclude-pattern>
+
+    <!-- How to scan -->
+    <arg value="sp"/> <!-- Show sniff and progress -->
+    <arg name="basepath" value="."/><!-- Strip the file paths down to the relevant bit -->
+    <arg name="colors" />
+    <arg name="extensions" value="php"/>
+    <arg name="report" value="full"/>
+    <arg name="report" value="summary"/>
+    <arg name="report" value="source"/>
+
+	<!-- Rules: Check PHP version compatibility -->
+	<config name="testVersion" value="5.3-"/>
+	<rule ref="PHPCompatibility"/>
```

Comment:
Since this is a WordPress plugin/theme, then it can reference the `PHPCompatibilityWP` ruleset instead - would need the `composer.json` template updated as well to pull that package in.

Reply:
Since that is outstanding, should this remain as is and implement a change for it with #63?

Reply:
I would most definitely recommend changing this already to use the `PHPCompatibilityWP` standard instead as it will prevent false positives for things WP back-fills/poly-fills.